docs/eloquent-models/model-class.txt

```diff
@@ -194,6 +194,24 @@ type, to the Laravel ``datetime`` type.
    :emphasize-lines: 9-11
    :dedent:
 
+.. tip:: Casts in Laravel 11
+
+   In Laravel 11, you can define a ``casts()`` method to specify data type conversions
+   instead of using the ``$casts`` attribute. The following code converts
+   the ``$casts`` attribute in the ``Planet`` model class to a ``casts()`` method:
```

Comment:
```suggestion
   instead of using the ``$casts`` attribute. The following code performs
   the same conversion as in the preceding example by using a ``casts()`` method:
```